lib/devise/multi_email/parent_model_manager.rb

```diff
@@ -62,7 +62,7 @@ def emails
 
       # Gets the email records that have not been deleted
       def filtered_emails(options = {})
-        emails.to_a.reject(&:destroyed?).reject(&:marked_for_destruction?)
+        emails.reject(&:destroyed?).reject(&:marked_for_destruction?)
```

Comment:
Just curious, will `to_a` cause any problems?  I can't remember why it's `to_a` before.

Reply:
hmmm, let me add it back and run my tests (in another project) and see if it fails...

Reply:
Ah, it seems my tests pass with or without it.

In the old days Rails association collection doesn't have reject. But it does now.

Reply:
I see. Then still better `to_a` to keep the compatibility to the old version of rails.